src/replay/service.zig

```diff
@@ -2,27 +2,32 @@ const std = @import("std");
 const sig = @import("../sig.zig");
 const replay = @import("lib.zig");
 
-const Allocator = std.mem.Allocator;
+const collections = sig.utils.collections;
 
+const Allocator = std.mem.Allocator;
 const ThreadPool = sig.sync.ThreadPool;
 
-const AccountStore = sig.accounts_db.account_store.AccountStore;
-const SlotAccountReader = sig.accounts_db.account_store.SlotAccountReader;
```

Comment:
I noticed you remove a lot of imports and replace them with fully qualified usages throughout the code. I think the code is clearer when each file lists its dependencies on top. I'd rather not have fully qualified usages littered throughout the code because it's not as easily discoverable.

Reply:
> the sheer mass of imports in many files does not give me any extra context I can work with

It definitely gives me context, especially if it's well organized.

We can leave the code as-is for now. Importing style is not our top priority, and with you pivoting to the transaction harness I'd just like to get this merged. We have a lot of disagreement about how to import, which means we need to spend a while learning about each other's values. Ultimately I think we should standardize our approach, but we can revisit it later to make a decision.